syntax = "proto3";

message Pizza {
    repeated string topping = 1;
    int32 pulgadas = 2;
}

message Orden {
    repeated Pizza pizzas = 1;
    string direccion = 2;
}

message ConfirmacionOrden {
    int64 entrega_estimada = 1;
}

message Nulo {
}

service Pizzeria {
    rpc Listo(Nulo) returns (Nulo);
    rpc RegistraOrden(Orden) returns (ConfirmacionOrden);
}
